### DISCUSSION TITLE
Help to import certificates

### DISCUSSION THREAD

**Discussion by rafaelhecr** [alexandercerutti/passkit-generator#97] [Q&A] [answered]
Hello! I'm Rafael and I'm trying to run this library, is great!, but i have throubles importing certificates, I'm using fs.readFileSync:
<img width="511" alt="Captura de Pantalla 2021-12-23 a la(s) 12 48 05" src="https://user-images.githubusercontent.com/19279811/147280951-98da9980-8ffb-43b9-93b2-877d867709f3.png">

but throw this error in terminal:
<img width="576" alt="Captura de Pantalla 2021-12-23 a la(s) 12 48 46" src="https://user-images.githubusercontent.com/19279811/147281081-fa214538-cb4c-4c1e-b568-aad85b02dd5d.png">

Please, can you tell me how import the binary files?.

Thank you.

**Comment by alexandercerutti** [comment, marked as answer]
Hey there, @rafaelhecr, thanks for using passkit-generator!
What I can suggest to you, is to check the value of `__dirname` before returning and using `path.resolve` with it to get certificates.
Also, you might want to try to print the value of what has been read through `fs.readFileSync` before returning.

Also, you might want to set up a way to set your data as strings in `process.env`.

Let me know!

**Reply by rafaelhecr** [reply]
Hello again, works using `__dirname`

<img width="606" alt="Captura de Pantalla 2021-12-23 a la(s) 13 28 09" src="https://user-images.githubusercontent.com/19279811/147284373-43829a99-3110-4e06-a8ff-4a58e460a0f7.png">

Thanks a lot!

**Reply by alexandercerutti** [reply]
It is more readable if you do like:

```javascript
fs.readFileSync(path.resolve(__dirname, "signerCert.pem");
```

😉
